engine/image-loader/image.lisp

```diff
@@ -0,0 +1,24 @@
+(in-package :lbge.image-loader.image)
```

Comment:
Hmm...
I don't see this file added to `lbge.asd`, and also I don't see where you define package `lbge.image-loader.image`.
I suppose this is the reason why your `image-test.lisp` unit test failed.

Reply:
Ah found the package definition, nvm.